V1/Frontend/src/screens/Home.jsx

```diff
@@ -0,0 +1,226 @@
+import React, { useState, useEffect } from "react";
+import Footer from "../components/Footer";
+import Navbar from "../components/Navbar";
+import logo from '../Assets/logo.png';
+import InfiniteLogo from "../components/InfiniteLogo";
+import { useLocation } from 'react-router-dom';
+import GraphComponent from "../components/Graph";
+
+export default function Home() {
+  const [searchText, setSearchText] = useState('');
+  const [results, setResults] = useState([]);
+  const [GraphData, setGraph] = useState([]);
+  const [filteredResults, setFilteredResults] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [tableRendered, setTableRendered] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [selectedStatus, setSelectedStatus] = useState('');
+  const [searched, setSearched] = useState(false);
+  const location = useLocation(); 
+  useEffect(() => {
+    const searchParams = new URLSearchParams(location.search);
+    const searchTextParam = searchParams.get('searchText');
+    if (searchTextParam) {
+      setSearchText(searchTextParam);
+      // fetchData();
+    }
+  }, [location.search]); 
+
+  const fetchData = async () => {
+    setIsLoading(true);
+    try {
+      const response = await fetch(`https://pypirecom-api.azurewebsites.net/search?Search_Text=${searchText}`);
```

Comment:
Api url to be added in constants file

Reply:
done created a js file to store constant value for the BASE URL